scala/src/main/scala/smile/validation/ClusterMeasure.scala

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
You don't need to define `ClusterMeasure`. It is already defined in Java and Scala can implement a Java interface.